spikesorters/sorterlist.py

```diff
@@ -152,7 +157,11 @@ def run_hdsort(*args, **kwargs):
             If True, output is verbose
         raise_error: bool
             If True, an error is raised if spike sorting fails (default). If False, the process continues and the error 
-            is logged in the log file.
+            is logged in the log file
+        n_jobs: int
+            Number of jobs when parallel=True (default=-1)
+        joblib_backend: str
+            joblib backend when parallel=True (default='loky')
     **kwargs: keyword args
```

Comment:
We coule have this 4 lines of docs somewhere as a string and add it to each run_XXXx this would avoid this ugly copy/paste.

Reply:
how?

Reply:
You could do something like that:
```python

common_doc = """
Alessio a fait une belle coupe de cheveux avant d'aller en Italie.
"""

def func1(...):
   """
   Car il va voir sa maman, 
   """  + common_doc

def func(...):
   """
   Car il va voir sa chérie, 
   """  + common_doc
```

Reply:
this is not possible, unfortunately. You will have to live with this ugly copy/paste ;)